test/kpod.bats

```diff
@@ -116,7 +116,7 @@ function teardown() {
 @test "kpod history json flag" {
 	run ${KPOD_BINARY} ${KPOD_OPTIONS} pull $IMAGE
 	[ "$status" -eq 0 ]
-	run bash -c "${KPOD_BINARY} ${KPOD_OPTIONS} history --json $IMAGE | python -m json.tool"
+	run bash -c "${KPOD_BINARY} ${KPOD_OPTIONS} history --format json $IMAGE | python -m json.tool"
 	echo "$output"
```

Comment:
It would be nice to change the name of the test on line 76 from 

kpod history with format

to

kpod history with go template